docs/config-files/config.yml

```diff
@@ -0,0 +1,23 @@
+# YAML config
+mode: # key
```

Comment:
This seems a little odd construction you have.  I would think that the config should look something more like:
```
mode:
 linear sweep voltammetry:
     start_voltage_mv: 10
     end_voltage_mv: 20
     step_size_mv: 1 
     hold_time_s: 0.1
 cyclic voltammetry:
    low_voltage_mv: 
    ...
```